x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/CcrIntegTestCase.java

```diff
@@ -487,6 +488,9 @@ protected void assertIndexFullyReplicatedToFollower(String leaderIndex, String f
             }
             IndexShard indexShard = cluster.getInstance(IndicesService.class, state.nodes().get(shardRouting.currentNodeId()).getName())
                 .indexServiceSafe(shardRouting.index()).getShard(shardRouting.id());
+            if (indexShard.state() != IndexShardState.STARTED) {
+                continue;
+            }
```

Comment:
I think we should catch "AlreadyClosedException" when calling `IndexShardTestCase.getDocIdAndSeqNos(indexShard)` instead?